src/Monticello/MCMethodDefinition.class.st

```diff
@@ -367,7 +352,12 @@ MCMethodDefinition >> shortSummaryPrefix [
 
 { #category : #comparing }
 MCMethodDefinition >> sortKey [
-	^ self className, '.', (self classIsMeta ifTrue: ['meta'] ifFalse: ['nonmeta']), '.', self selector
+	^ sortKey
+		ifNil: [ sortKey := self className , '.'
+				,
+					(self classIsMeta
+						ifTrue: [ 'meta' ]
+						ifFalse: [ 'nonmeta' ]) , '.' , self selector ]
```

Comment:
Maybe we could even have:

```Smalltalk
self className, (self classIsMeta ifTrue: ['.meta.'] ifFalse: ['.nonmeta.']), self selector
```

to avoid useless concatenations.